infrastructure/material_list/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
```

Comment:
This includes many different options which does not seem relevant to the current project. Should they be removed?

Reply:
It's the default helm generates....